src/components/NoticeBox.js

```diff
@@ -1,17 +1,64 @@
 import React from "react";
```

Comment:
General note for all the components - we usually use a `.jsx` file suffix for any React/Preact component file rather than `.js`.

Reply:
Ahh, good to know! Thanks! 👍